airflow/models/dag.py

```diff
@@ -1506,6 +1506,12 @@ def safe_dag_id(self):
         return self.dag_id.replace('.', '__dot__')
 
     def get_dag(self):
+        try:
+            path_split = self.fileloc.split("airflow_home")[1]
```

Comment:
This seems a bit brittle, if someone creates a dir called airflow_home in their dag dir it will cause it to break. Curious why we can't split on the whole $AIRFLOW_HOME rather than just "airflow_home" (which can also break if we change the name of our airflow home dir)

Reply:
The issue here is that webserver is getting the scheduler path(for dag), and since that path doesn't exist in webserver, manual trigger breaks.
In order to fix this, I have put this hack. I think we can replace it with regex, that should be good.